### PR TITLE
getting-started: update: change to v1.5.1

### PR DESCRIPTION
**Pull request by lucasdemarchi**

### CODE REVIEW

File: docs/getting-started/update.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ and use the following command on the terminal:
 
 ``` console
 # aero-get-version.py
-BIOS_VERSION = Aero-01.00.12_Prod
-OS_VERSION = v1.4
+BIOS_VERSION = Aero-01.00.13_Prod
```
Comment (zehortigoza) [inline docs/getting-started/update.md]: At least in this BIOS release there is no _Prod, so it should only be Aero-01.00.13

File: docs/getting-started/update.md [hunk 2]
```diff
@@ -193,26 +187,40 @@ The updated process gives feedback to the user in some forms:
...
 ``` console
-$ scp ~/Downloads/capsule-01.00.12-r0.core2_64.rpm intel-aero.local:
+# aero-bios-update
+Bios update set up to take place on next boot
+It's recommended to have a monitor plugged on hdmi
+to follow update progress.
 ```
 
-1. Install and reboot
+During the next boot the BIOS will detect the update and flash the new version.
```
Comment (zehortigoza) [inline docs/getting-started/update.md]: Maybe add some warning to user dot turn the board off during the BIOS update.

Comment (lucasdemarchi) [inline docs/getting-started/update.md]: Done

File: docs/getting-started/update.md (hunk 1 above)
Comment (lucasdemarchi) [inline docs/getting-started/update.md]: Thanks, I hadn't noticed. Fixed.

**Pull request merged**